src/compression/compress_h5mu/run_test.py

```diff
@@ -0,0 +1,51 @@
+
+import sys
+import pytest
+import mudata as mu
+from pathlib import Path
+import pandas as pd
+
+## VIASH START
+meta = {
+    'executable': './target/docker/compression/compress_h5mu/compress_h5mu',
+    'resources_dir': 'resources_test/concat_test_data/',
+    'config': 'src/compression/compress_h5mu/config.vsh.yaml'
+}
+## VIASH END
+
+
+input_file = Path(f"{meta['resources_dir']}/e18_mouse_brain_fresh_5k_filtered_feature_bc_matrix_subset_unique_obs.h5mu")
+
+def compare_anndata(first, second):
+    for attr_name in ("obs", "var"):
+        pd.testing.assert_frame_equal(getattr(first, attr_name), 
+                                      getattr(second, attr_name))
```

Comment:
Seems like this could be replaced with `from anndata.tests.helpers import assert_equal`, correct?